### DISCUSSION TITLE
Can't get proper API response

### DISCUSSION THREAD

**Discussion by Natooz** [navidrome/navidrome#2380] [Q&A] [answered]
Hello,

First thanks for your hard work!

I am currently trying to use the API, but the server doesn't return the expected response format.
I don't think I'm sending request with bad format: `http://192.168.1.1:4533/rest/getGenres?c=name&f=json&s=TEST&t=MYTOKEN&u=user&v=1.15.0` 

The server always returns an html result with the node.js error *You need to enable JavaScript to run this app.*.

Any idea of what I am doing wrong ?

**Comment by deluan** [comment, marked as answer]
The URL seems correct. The message you see is that your request is not reaching the correct endpoint, it thinks it is a URL for the WebUI app. What is the URL you usually use in your browser to access the UI?

**Reply by Natooz** [reply]
You are right!
As I setup the server a while now, I forgot I put a `BASE_URL`. It's working perfectly now, thanks! :)
